pkg/arvo/app/landscape/js/channel.js

```diff
@@ -210,6 +212,9 @@ class Channel {
         if (obj.hasOwnProperty("err")) {
           funcs["err"](obj.err);
           this.outstandingSubscriptions.delete(obj.id);
+        } else if (obj.hasOwnProperty("ok")) {
+          funcs["subAck"](obj);
+          this.outstandingSubscriptions.delete(obj.id);
```

Comment:
I don't think we want to delete the subscription when we get an ack for it, right? We still want to have its `event` callback around, and maybe it can error out later.